gui/CourseGeneratorScreen.lua

```diff
@@ -548,6 +551,71 @@ function CourseGeneratorScreen:onClickNumberOfRowsPerLand(state)
 	end
 end
 
+function CourseGeneratorScreen:onOpenShowSeedCalculator( element, parameter )
+	local setting = self.vehicle.cp.courseGeneratorSettings.showSeedCalculator
+	setting:setGuiElement(element)
+	element:setTexts(setting:getGuiElementTexts())
+	element:setState(setting:getGuiElementState())
+end
+
+function CourseGeneratorScreen:onClickShowSeedCalculator(state)
+	local setting = self.vehicle.cp.courseGeneratorSettings.showSeedCalculator
+	if setting:getGuiElement() then
+		setting:setToIx(setting:getGuiElement():getState())
+	end
+end
+
+---a very basic and simple seed calculator in the course generator
+function CourseGeneratorScreen:drawSeedCalculator(xPos,yPos)
+	-- do have a valid field selected ?
+	local currentFieldNumber = self.vehicle.cp.fieldEdge.selectedField.fieldNum
+	if currentFieldNumber ~= 0 then 
+		local fieldAreaHa = courseplay.fields.fieldData[currentFieldNumber].areaHa
+		local fieldAreaSqm = courseplay.fields.fieldData[currentFieldNumber].areaSqm
+		setTextBold(true)
+		local textFontSize = 0.02
+		local shadowOffset = textFontSize * 0.03
+		--shadow color
+		local rShadow,gShadow,bShadow,aShadow = 1, 0, 0, 0.8
+		--text color
+		local r,g,b,a = 1, 0.2, 0, 1
+		-- draw shadow
+		setTextColor(rShadow,gShadow,bShadow,aShadow)
+		renderText(self.ingameMap.absPosition[ 1 ]+shadowOffset,self.ingameMap.absPosition[ 2 ]-shadowOffset,textFontSize,string.format("Field size: %.2f Ha",fieldAreaHa))
+		-- draw field size at the bottom
+		setTextColor(r,g,b,a)
+		renderText(self.ingameMap.absPosition[ 1 ],self.ingameMap.absPosition[ 2 ],textFontSize,string.format("Field size: %.2f Ha",fieldAreaHa))
+		-- draw all the sprayTypes and fruitType  
+		for _,sprayType in pairs(g_sprayTypeManager:getSprayTypes()) do
+			local litersPerSecond = sprayType.litersPerSecond
+			local totalLiters = litersPerSecond*fieldAreaHa*36000
```

Comment:
Not saying this is wrong, just wondering how is this calculated, where the 36000 is coming from (a comment would have been nice)